plugins/org.locationtech.udig.libs/pom-libs.xml

```diff
@@ -604,95 +569,36 @@
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-gvt</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-ext</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-xml</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-script</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-awt-util</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.xmlgraphics</groupId>
-			<artifactId>batik-parser</artifactId>
-			<version>1.7</version>
-			<scope>provided</scope>
-		</dependency>
-
-		<!-- These dependencies are for optional batik components and are not used -->
-		<dependency>
-			<groupId>xml-apis</groupId>
-			<artifactId>xml-apis-xerces</artifactId>
-			<version>2.7.1</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>xml-apis</groupId>
-			<artifactId>xml-apis</artifactId>
-			<version>1.0.b2</version>
-			<scope>provided</scope>
+			<exclusions>
+				<!-- batik jars provided as plugin by target platform, see import-packages and Required-Bundles in Manifest-File-->
+				<!--
+					+- org.geotools:gt-svg:jar:22.1:compile
+					|  \- org.apache.xmlgraphics:batik-transcoder:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-anim:jar:1.10:compile
+					|     |  +- org.apache.xmlgraphics:batik-css:jar:1.10:compile
+					|     |  +- org.apache.xmlgraphics:batik-ext:jar:1.10:compile
+					|     |  +- org.apache.xmlgraphics:batik-parser:jar:1.10:compile
+					|     |  \- org.apache.xmlgraphics:batik-svg-dom:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-awt-util:jar:1.10:compile
+					|     |  \- org.apache.xmlgraphics:xmlgraphics-commons:jar:2.2:compile
+					|     +- org.apache.xmlgraphics:batik-bridge:jar:1.10:compile
+					|     |  \- org.apache.xmlgraphics:batik-script:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-dom:jar:1.10:compile
+					|     |  \- xalan:xalan:jar:2.7.2:compile
+					|     |     \- xalan:serializer:jar:2.7.2:compile
+					|     +- org.apache.xmlgraphics:batik-gvt:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-svggen:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-util:jar:1.10:compile
+					|     |  +- org.apache.xmlgraphics:batik-constants:jar:1.10:compile
+					|     |  \- org.apache.xmlgraphics:batik-i18n:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-xml:jar:1.10:compile
+					|     \- xml-apis:xml-apis-ext:jar:1.3.04:compile
+				-->

```

Comment:
What is this comment useful for?

Reply:
it helps to understand which transitive dependencies are not bundled again and might be resolved via "Import-Package" or "Required-Bundle" in MANIFEST.MF from the OSGi platform. just kind of documentation ;)

Reply:
If you think this is worth to put into the pom-libs.xml I am fine with this PR 👍